### PR TITLE
Add assignment1

### PR DESCRIPTION
**Pull request by ashneeldas2**

### CODE REVIEW

File: versioned_docs/version-2020fa/assignments.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ title: Assignments
 ---
 
 Assignments will be released here after lecture!
+
+[Assignment 1](./assignment1): Due 9/29 before class
```
Comment (meganyin13) [inline versioned_docs/version-2020fa/assignments.md:8]: Could we clarify this to `Due 9/29 at 3:59pm on CMS`

**Review by meganyin13 — COMMENTED** [review]
Thanks Ashneel! Could we also add some submission instructions to the end of the A1 writeup like: please submit to CMS the `index.ts` file where you have defined your routes by Tuesday 9/29 3:59pm.

**Review by SamChou19815 — COMMENTED** [review]
I think we need some instructions on how to setup the project.

e.g.
```
yarn init -y
yarn add express
yarn add --dev typescript ts-node @types/express
```

and an example tsconfig.json

**Comment by meganyin13** [comment]
> I think we need some instructions on how to setup the project.
> 
> e.g.
> 
> ```
> yarn init -y
> yarn add express
> yarn add --dev typescript ts-node @types/express
> ```
> 
> and an example tsconfig.json

Ideally, they should know setup if they took a look at lecture 1, but I've added the yarn commands for setup anyway. I didn't talk about `tsconfig.json` yet (will do so next lecture) so we'll just use ts-node directly. (I also forgot to say typescript, ts-node are devDependencies in the new recordings but that should be ok 😅) Thanks Sam!

**Comment by SamChou19815** [comment]
> > I think we need some instructions on how to setup the project.
> > e.g.
> > ```
> > yarn init -y
> > yarn add express
> > yarn add --dev typescript ts-node @types/express
> > ```
> > 
> > 
> > and an example tsconfig.json
> 
> Ideally, they should know setup if they took a look at lecture 1, but I've added the yarn commands for setup anyway. I didn't talk about `tsconfig.json` yet (will do so next lecture) so we'll just use ts-node directly. (I also forgot to say typescript, ts-node are devDependencies in the new recordings but that should be ok 😅) Thanks Sam!

One potential issue is that the "strict" flag is by default off. Without `tsconfig.json`, you can never configure `strict`, so errors related to null or undefined will not be shown. I guess it's fine this time.

**Pull request merged**